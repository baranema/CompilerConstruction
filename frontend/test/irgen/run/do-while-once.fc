extern int printf(char[] format, ...);

int main() {
    int i = 3;

    do {
        printf("print me pretty please\n");
    } while(1 == 0);

    printf("done\n");
    return 0;
}
