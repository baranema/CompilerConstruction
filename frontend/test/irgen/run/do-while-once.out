print me pretty please
done
